### PR TITLE
Skip elements with titles.

### PR DESCRIPTION
**Pull request by dblock**
### Description

A workaround for the change in https://github.com/opensearch-project/opensearch-api-specification/pull/416 to avoid adding `node_id_or_metric`. With https://github.com/opensearch-project/opensearch-api-specification/pull/434 the only API tests that need to be skipped are those that `catch` an error (missing index parameters). These fail as expected on the client side.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by saimedhi** [comment]
@dblock, Please take a look at the failing tests. Thank you.

**Comment by dblock** [comment]
The failure is unrelated caused by https://github.com/opensearch-project/OpenSearch/commit/29a3e2c980764f305f0ed5e858878bc7bb3dbe64. The `put_alias` method doesn't actually require index or alias, which can be specified in the body. 

What's the right way to fix this @saimedhi?

**Comment by saimedhi** [comment]
> The failure is unrelated caused by [opensearch-project/OpenSearch@29a3e2c](https://github.com/opensearch-project/OpenSearch/commit/29a3e2c980764f305f0ed5e858878bc7bb3dbe64). The `put_alias` method doesn't actually require index or alias, which can be specified in the body. The tests added in that PR do that.
> 
> What's the right way to fix this @saimedhi? Where do I remove the fact that these parameters are required and can default to `None`?

Hi @dblock, for path params they can be changed [here](https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate_api.py#L386)

**Comment by dblock** [comment]
https://github.com/opensearch-project/opensearch-py/pull/778

**Review by saimedhi — APPROVED** [review]

File: opensearchpy/client/indices.py [hunk 1]
```diff
@@ -861,23 +864,23 @@ def get_field_mapping(
     )
     def put_alias(
         self,
-        index: Any,
-        name: Any,
         body: Any = None,
+        index: Any = None,
+        name: Any = None,
```
Comment (Tenzer) [inline opensearchpy/client/indices.py:869]: FYI: This is a breaking change for anybody that were calling the method without naming the arguments.

Comment (dblock) [reply opensearchpy/client/indices.py:869]: Looks like a miss :(

Care to open a bug, or if you have time fix the code generator?

Comment (Tenzer) [reply opensearchpy/client/indices.py:869]: The number of users affected by this is probably limited, but perhaps it's worth calling it out in the release notes?

Comment (Tenzer) [reply opensearchpy/client/indices.py:869]: I have made a bug for it here: https://github.com/opensearch-project/opensearch-py/issues/803.

**Pull request merged**